code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -117,37 +117,25 @@
 	robotize()
 	..()
 
-/obj/item/organ/internal/optical_sensor
+/obj/item/organ/internal/eyes/optical_sensor
```

Comment:
Just addressed this with [this commit](https://github.com/ParadiseSS13/Paradise/pull/6123/commits/49a0aac88aa447a341bad535f2300331f0ad0096), thanks for being specific about it

Tested by welder-spamming a wall as an IPC, no damage to eyes as before